pages/settings/about/index.vue

```diff
@@ -17,7 +17,7 @@ function handleShowCommit() {
 </script>
 
 <template>
-  <MainContent back-on-small-screen>
+  <MainContent back-on-small-screen no-beta-label>
```

Comment:
Another Vue nuance!! Idk why the [recommendation](https://v2.vuejs.org/v2/style-guide/#Prop-name-casing-strongly-recommended) to declare props in camelCase, but then use them in kebab-case in the HTML bugs me SOOOOOO much, but it does.